### PR TITLE
linked the wrong thing. Fixing it

### PR DESCRIPTION
**Pull request by arrowarchive**

### CODE REVIEW

File: docs/web/scaling.html [hunk 1]
```diff
@@ -10,9 +10,9 @@
         <div class="flex">
             <div class="sidebar">
                 <div>
-                    <img src="images/assets/Aqua.png" alt="profile picture"
+                    <a href="https://arrowarchive.github.io/The-Arrowarchive/index.html"><img src="images/assets/Aqua.png" alt="profile picture"
```
Comment (am0eba-byte) [inline docs/web/scaling.html:13]: very linky

File: docs/web/scaling.html [hunk 2]
```diff
@@ -27,7 +27,7 @@
             </div>
             <div class="main">
         <h1>Improving the Archive: part 1</h1>
-                <a href="https://arrowarchive.github.io/The-Arrowarchive/index.html"><img src="images/unravelPlaceholder.png" width="627" oncontextmenu="return false;"/></a>
```
Comment (ebeshero) [inline docs/web/scaling.html:30]: @arrowarchive Nice catch here with repairing this broken link.

File: docs/web/scaling.html [hunk 3]
```diff
@@ -27,7 +27,7 @@
             </div>
```
Comment (Tiny-Pickles) [inline docs/web/scaling.html:27]: cool code!

File: docs/web/scaling.html (hunk 1 above)
Comment (Yuying-Jin) [reply docs/web/scaling.html:13]: HA!

**Pull request merged**